src/dune_rules/context.ml

```diff
@@ -313,8 +313,7 @@ let ocamlfind_printconf_path ~env ~ocamlfind ~toolchain =
   in
   let+ l =
     Memo.of_reproducible_fiber
-      (Process.run_capture_lines ~display:!Clflags.display ~env Strict ocamlfind
-         args)
+      (Process.run_capture_lines ~display:Quiet ~env Strict ocamlfind args)
```

Comment:
Here we query the configuration of ocamlfind. This is internal and does not need to be reported to the user.